recipes/http-ece/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "http-ece" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/http_ece-{{ version }}.tar.gz
+  sha256: 932ebc2fa7c216954c320a188ae9c1f04d01e67bec9cdce1bfbc912813b0b4f8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Sorry, I missed these.

```suggestion
    - python >=3.6
```